bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/AccountHandler.java

```diff
@@ -709,28 +709,35 @@ public List<SmartHomeBaseDevice> updateSmartHomeDeviceList(boolean forceUpdate)
             return List.of();
         }
 
+        String jsonQuery = "{\"query\":\"query Endpoints{endpoints{items{endpointId id friendlyName displayCategories{primary{value}} legacyIdentifiers{dmsIdentifier{deviceType{type value{text}} deviceSerialNumber{type value{text}}}} legacyAppliance{applianceId applianceTypes endpointTypeId friendlyName friendlyDescription manufacturerName connectedVia modelName entityId actions mergedApplianceIds capabilities applianceNetworkState version isEnabled customerDefinedDeviceType customerPreference alexaDeviceIdentifierList aliases driverIdentity additionalApplianceDetails isConsentRequired applianceKey appliancePairs deduplicatedPairs entityPairs deduplicatedAliasesByEntityId relations} serialNumber{value{text}} enablement model{value{text}} manufacturer{value{text}} features{name operations{name}}}}}\"}";
+
         try {
             if (connection.isLoggedIn()) {
-                JsonNetworkDetails networkDetails = connection.getRequestBuilder()
-                        .get(connection.getAlexaServer() + "/api/phoenix").syncSend(JsonNetworkDetails.class);
-                Object jsonObject = gson.fromJson(networkDetails.networkDetail, Object.class);
-                List<SmartHomeBaseDevice> smartHomeDevices = new ArrayList<>();
-                searchSmartHomeDevicesRecursive(jsonObject, smartHomeDevices);
+                SmartHomeTO networkDetails = connection.getRequestBuilder()
+                        .post(connection.getAlexaServer() + "/nexus/v1/graphql").withContent(jsonQuery).withJson(true)
+                        .syncSend(SmartHomeTO.class);
 
-                // create new id map
+                List<JsonSmartHomeDevice> smartHomeDevices = (networkDetails != null)
+                        ? networkDetails.getLegacySmartHomeDevices()
+                        : List.of();
                 Map<String, SmartHomeBaseDevice> newJsonIdSmartHomeDeviceMapping = new HashMap<>();
-                for (SmartHomeBaseDevice smartHomeDevice : smartHomeDevices) {
+                for (JsonSmartHomeDevice smartHomeDevice : smartHomeDevices) {
                     String id = smartHomeDevice.findId();
                     if (id != null) {
                         newJsonIdSmartHomeDeviceMapping.put(id, smartHomeDevice);
                     }
                 }
+                // TODO previously `searchSmartHomeDevicesRecursive` was called here. Due to changes in the JSON
```

Comment:
To the reviewer: The groups seem to be missing from the response. I'll continue to investigate and maybe with some testers we can find a fix. 
Anyway, this PR should not wait for this fix to be merged as currently this is fully broken.